### PR TITLE
Vue Testing 

### PR DESCRIPTION
**Pull request by andrewrady**

### CODE REVIEW

File: _posts/2020-03-06-testing-components-vue-conf-2020.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Getting Started Testing Vue Components"
+date: 2020-03-06 10:43:25
+tags: 
+- Vue.js
+- Jest
+- Testing
+categories:
+twitter_text: Getting started with testings your Vue components with Jest
```
Comment (kendaleiv) [inline _posts/2020-03-06-testing-components-vue-conf-2020.md]:
```suggestion
twitter_text: Getting started with testing your Vue components with Jest
```

File: _posts/2020-03-06-testing-components-vue-conf-2020.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+authors: Andrew Rady
+image: https://images.unsplash.com/photo-1560264280-88b68371db39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80
+image_url: https://unsplash.com/photos/kN_kViDchA0
+image_credit: Arlington Research
+---
+
+Testing is an important step we can take as developer to reduce bugs in our code, but testing seems to be one of the best practices that is skipped the most. When it comes to development I have noticed that testing is its own framework all in itself that you need to learn, and understand how it works. If you have worked on a large scale application you know how hard manual testing becomes. As front end developers who write with Vue.js Jest is one of the recommended testing framework. Let's dive into using Jest the [vue test utils](https://www.npmjs.com/package/@vue/test-utils) package in our applications.
```
Comment (kendaleiv) [inline _posts/2020-03-06-testing-components-vue-conf-2020.md]:
```suggestion
Testing is an important step we can take as developers to reduce bugs in our code, but testing seems to be one of the best practices that is skipped the most. When it comes to development I have noticed that testing is its own framework all in itself that you need to learn, and understand how it works. If you have worked on a large scale application you know how hard manual testing becomes. As front end developers who write with Vue.js Jest is one of the recommended testing framework. Let's dive into using Jest the [vue test utils](https://www.npmjs.com/package/@vue/test-utils) package in our applications.
```

File: _posts/2020-03-06-testing-components-vue-conf-2020.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+  wrapper.setData({
+    showDetails: true
+  })
+
+  wrapper.vm.increase()
+  
+  expect(wrapper.vm.amount).toBe(1)
+})
+```
```
Comment (kendaleiv) [inline _posts/2020-03-06-testing-components-vue-conf-2020.md]: Want to add a summary or closing remark?

Comment (andrewrady) [reply _posts/2020-03-06-testing-components-vue-conf-2020.md]: Add a conclusion, thanks!

**Pull request merged**